docs/patterns/single-new-eks-observability-accelerators/existing-eks-mixed-observability.md

```diff
@@ -0,0 +1,39 @@
+# Existing EKS Cluster AWS Mixed Observability Accelerator
+
+## Objective
+
+This pattern depends on the previous deployment of [_Single New EKS Cluster Observability Accelerator_](./single-new-eks-cluster.md).
```

Comment:
Objective is not spelled out correctly. Objective is to create observability with a mixture of AWS native and Open source managed AWS services on an existing EKS cluster. Also talk about what this mixed observability provides to single new mixed doc.